### PR TITLE
add rc test for ASM_DD rules

### PR DESCRIPTION
**Pull request by christophe-papazian**
## Motivation

When ASM_DD stop sending a security rule file, the tracer must switch back to the default static file configuration.

## Changes

Add a remote config test for that behaviour.

APPSEC-54105

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by Anilm3 — APPROVED** [review]
LGTM, thanks @christophe-papazian!

File: manifests/php.yml [hunk 1]
```diff
@@ -181,6 +181,7 @@ tests/:
       Test_StandardizationBlockMode: missing_feature
     test_remote_config_rule_changes.py:
       Test_BlockingActionChangesWithRemoteConfig: missing_feature
+      Test_UpdateRuleFileWithRemoteConfig: missing_feature
```
Comment (Anilm3) [inline manifests/php.yml:184]: Should work since `v0.8.0` if you want to update it directly.

Comment (christophe-papazian) [reply manifests/php.yml:184]: I'll do it. Waiting for more feedbacks...

**Review by pawelchcki — APPROVED** [review]

File: manifests/php.yml (hunk 1 above)
Comment (christophe-papazian) [reply manifests/php.yml]: done

File: manifests/nodejs.yml [hunk 2]
```diff
@@ -355,6 +355,7 @@ tests/:
       Test_Main: v2.0.0
     test_remote_config_rule_changes.py:
       Test_BlockingActionChangesWithRemoteConfig: *ref_4_1_0
+      Test_UpdateRuleFileWithRemoteConfig: missing_feature
```
Comment (simon-id) [inline manifests/nodejs.yml:358]: same version as when ASM_DD was implemented
```suggestion
      Test_UpdateRuleFileWithRemoteConfig: *ref_3_19_0
```

**Pull request merged**